src/dataflow/components/dataflow-program.tsx

```diff
@@ -163,6 +151,10 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
         hubStore.hubs.forEach(hub => {
           hub.hubChannels.forEach(ch => {
             const chValue = Number.parseFloat(ch.value);
+            const nci = this.channels.find(ci => ci.hubId === hub.hubId && ci.channelId === ch.id);
+            if (nci && !Number.isNaN(chValue)) {
```

Comment:
```suggestion
            if (nci && Number.isFinite(chValue)) {
```
`isFinite()` catches infinities as well as NaNs and makes code more readable by avoiding double-negatives -- "is this not not-a-number?" As a general rule I prefer `isFinite()` unless there's a particular reason to handle `infinity` differently than `NaN`.